Jenkinsfile

```diff
@@ -84,7 +84,30 @@ elifePipeline {
             }
         }
 
-        stage 'Browser Tests', {
+        stage 'Integration Tests', {
+            def actions = [
+                "browser": {
+                  sh "IMAGE_TAG=${commit} NODE_ENV=production NODE_CONFIG_ENV=test docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_wait-app-browser-test app bash -c './scripts/wait-for-app.sh app:3000'"
+                  withCommitStatus({
+                      sh "IMAGE_TAG=${commit} NODE_ENV=production NODE_CONFIG_ENV=test docker-compose -f docker-compose.yml -f docker-compose.ci.yml run -p 10081:10081 --rm --name elife-xpub_app_test_browser test_browser"
+                      sh "IMAGE_TAG=${commit} NODE_ENV=production NODE_CONFIG_ENV=unit-test docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_app_test app bash -c 'scripts/pipeline-log-filter-test.sh'"
+                  }, 'test:browser', commit)
+                },
+                "api": {
+                  sh "IMAGE_TAG=${commit} NODE_ENV=production NODE_CONFIG_ENV=test docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_wait-app-api-test app bash -c './scripts/wait-for-app.sh app:3000'"
+                  dir("tools/api-tests") {
+                    withCommitStatus({
+                      try {
+                        sh "docker ps"
```

Comment:
intentional not to use `docker-compose ps` which will limit the output to the containers to a subset?

Reply:
Oops - this was left in accidentally